src/app/App.js

```diff
@@ -1,61 +1,40 @@
 import React, { Component } from 'react';
 import { Route, Switch } from 'react-router-dom';
 import './App.css';
-import { fetchAllMovies, fetchMovieDetails } from '../APICalls'
+import { fetchAllMovies } from '../APICalls'
 import MovieContainer from '../container/MovieContainer'
 import MovieDetailsCard from '../details/MovieDetailsCard';
 
-class App extends Component {
+export default class App extends Component {
   constructor() {
     super()
     this.state = {
       movies: [],
-      displayMovieDetails: null,
-      libraryError: '',
-      movieDetailsError: ''
+      error: null
     }
   }
 
   render() {
-    return (
-      <main className="App">
-        <h1>Rancid Tomatillos</h1>
-        <Route exact path="/" render={() => <MovieContainer movies={this.state.movies} displayMovieDetails={this.displayMovieDetails} /> } />
-        <Route
-            path="/:id"
-            render={({ match }) => {
-              if (this.state.movies.length > 0) {
-                const { id } = match.params
-                let foundMovie = this.state.movies.find(movie => movie.id === parseInt(id))
-                return <MovieDetailsCard movie={foundMovie} displayMovieLibrary={this.displayMovieLibrary} error={this.state.movieDetailsError} />
-              }
-             }
-            }
-          />
-        {this.state.libraryError && <h2>{this.state.libraryError}</h2>}
-      </main>
-    )
-  }
-
-  displayMovieLibrary = () => {
-    this.setState({
-      displayMovieDetails: null
-    })
+    if (!this.state.error) {
+      return (
+        <main className="App">
+          <h1>Rancid Tomatillos</h1>
```

Comment:
I'm wondering if we may want to pull the H1 out of this conditional since it is the title of the app